modules/scala/install

```diff
@@ -6,6 +6,16 @@ SCRIPT_DIR=$(dirname $0)
 ADDED_DIR=${SCRIPT_DIR}/added
 ARTIFACTS_DIR=/tmp/artifacts
 
+fullname=$(find $ARTIFACTS_DIR -name spark-[0-9.]*-bin-hadoop[0-9.]*\.tgz)
```

Comment:
shouldn't the spark bits already exist since we're using the spark base image?

Reply:
Not really. We should use scala S2i as the base image, but we don't have a sclorg scala s2i image. That said, I need to check if we build an image under radanalyticsio namespace while waiting on sclorg scala s2i image.